rasa_core_sdk/forms.py

```diff
@@ -50,15 +50,20 @@ def from_intent(intent, value):
     def from_text(intent=None):
         return {"type": "from_text", "intent": intent}
 
-    def slot_mapping(self):
-        # type: () -> Dict[Text: Union[Text, Dict, List[Text, Dict]]]
+    # noinspection PyMethodMayBeStatic
+    def slots_mappings(self):
```

Comment:
i meant `slot_mappings` without the `s` at the end of slots 😄

Reply:
I know, but I thought since it is for many slots, it should be `slots_mappings`, no?